### DISCUSSION TITLE
Android version: Read a frequency file provided by the user

### DISCUSSION THREAD

**Discussion by lutusp** [AlexandreRouma/SDRPlusPlus#1359] [Ideas]
Just a suggestion for the future. The Android version works very well, but unlike the Windows/Linux/Mac versions, you can't simply replace the frequency_manager_config.json file with your own collection of frequencies, and the "import" button isn't active.

One suggestion would be to put a copy of a prepared frequency list in a location that SDR++ could read, example /storage/emulated/0/Android/data/org.sdrpp.sdrpp/files/. That location is easily written to by a sophisticated user with "adb", and SDR++ has the right to access it.

This is a bit ironic, but a modern Android tablet often has more computing power and storage space than many small systems like older Raspberry Pis, so it's a reasonable installation candidate -- all except the fact that you can't import frequencies.

Again, this is just a suggestion for the future, and SDR++ is a terrific program.

**Comment by TheFilthyFrog** [comment]
It's a shame we can't add our saved frequency_config from windows to android. Hopefully it's possible one day.
